ownCloud/Settings/ProPhotoUploadSettingsSection.swift

```diff
@@ -98,14 +109,20 @@ class ProPhotoUploadSettingsSection: SettingsSection {
 
 		self.add(row: preferOriginalsRow)
 
-		if AVCaptureDevice.rawCameraDeviceAvailable() {
```

Comment:
Why did you remove the `if` clause? Raw camera support is not available on every device.

Reply:
> Why did you remove the `if` clause? Raw camera support is not available on every device.

What about raw files stored on devices without the latest camera? I created the file in iPhone 12 Pro Max, but did my upload testing with the iPhone 8 test device.

Reply:
Ok, good point. I missed that option!

Reply:
@hosy that was exactly the intension to allow users upload RAW photos from their iCloud synced photo library shot on more capable device.